aws/code/workflows/MLWorkflowSingleCPU.py

```diff
@@ -55,7 +56,7 @@ def ingest_data(self):
         if 'Parquet' in self.hpo_config.input_file_type:
             hpo_log.info('> parquet data ingestion')
 
-            assert isinstance(self.hpo_config.target_files, str)
+            # assert isinstance(self.hpo_config.target_files, str)
```

Comment:
Any reason for removing this assert statement?

Reply:
I believe it was because when I was debugging, it was reading in a path object so the assertion was getting in the way. Also, https://github.com/sarahyurick/cloud-ml-examples/blob/kmeans/aws/code/workflows/MLWorkflowSingleGPU.py does not have any assert statements.